test/gui/shared/steps/steps.py

```diff
@@ -392,3 +395,22 @@ def step(context, resource, password):
     mouseClick(waitForObject(names.oCC_ShareLinkWidget_lineEdit_password_QLineEdit), 0, 0, Qt.NoModifier, Qt.LeftButton)
     type(waitForObject(names.oCC_ShareLinkWidget_lineEdit_password_QLineEdit), password)
     clickButton(waitForObject(names.oCC_ShareLinkWidget_createShareButton_QPushButton))
+
+
+@When('the user edits the public link named "|any|" of file "|any|" changing following')
+def step(context, publicLinkName, resource):
+    snooze(5)
```

Comment:
can we do without that static wait?

Reply:
I don't think there is another way for it.